server/templates/pages/src/index.html

```diff
@@ -38,15 +37,10 @@
 
         <a id="about-mozilla" rel="author" target="_blank" href="https://www.mozilla.org/about/?utm_source=firefox-accounts&amp;utm_medium=Referral"></a>
 
-        <!--[if lt IE 10]>
-            <p class="browsehappy">{{#t}}You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.{{/t}}</p>
```

Comment:
@philbooth @vbudhram thoughts on removing this? 

Can this be replaced with https://github.com/mozilla/fxa-content-server/issues/5558

Reply:
I don't think it can be replaced directly, since that issue is specific to upgrading Firefox version. 

From the looks of it, there doesn't appear to be any traffic coming from IE 10 and less browsers, https://app.datadoghq.com/dash/100285/fxa-playground?live=true&page=0&is_auto=false&from_ts=1520876900454&to_ts=1520963300454&tile_size=m&fullscreen=288019611, so it seems good to remove to me.

Reply:
yeah it's close to 0%